packages/sdk/base/src/phoneNumbers.ts

```diff
@@ -8,22 +6,8 @@ export interface ParsedPhoneNumber {
   regionCode?: string
 }
 
-const PHONE_SALT_SEPARATOR = '__'
 const E164_REGEX = /^\+[1-9][0-9]{1,14}$/
 
-export const getPhoneHash = (
-  sha3: (a: string) => string | null,
-  phoneNumber: string,
-  salt?: string
-): string => {
-  if (!phoneNumber || !isE164Number(phoneNumber)) {
```

Comment:
may be a good idea to add the `isE164Number` check for the PN special case in `getIdentifierHash`, to ensure that the identifiers are standardized + to prevent users from forgetting to format in this way!

alternatively this can also be added to the new `getPhoneHash`!

Reply:
We've added this check to getBlindedIdentifier https://github.com/celo-org/celo-monorepo/blob/2ef34baeabe3ae416970b3962df739a558d44dff/packages/sdk/identity/src/odis/identifier.ts#L176